src/main/java/frc/robot/subsystems/conveyor/ConveyorCal.java

```diff
@@ -15,7 +15,7 @@ public class ConveyorCal {
   public static final double SCORE_AMP_FRONT_SPEED = -0.5, SCORE_AMP_BACK_SPEED = -0.5;
 
   /** The power used for each conveyor motor to receive the note */
-  public static final double RECEIVE_SPEED = 1.0, RECEIVE_SLOW_SPEED = 0.15;
+  public static final double RECEIVE_SPEED = 1.0, RECEIVE_SLOW_SPEED = 0.25;
```

Comment:
Was this tested? Let's adjust this once we're testing

Reply:
Don't think so but I think we merge it for now